test/browser/LocalizationLoader.js

```diff
@@ -30,29 +30,30 @@
 
 'use strict';
 
-var assert = require('assert'),
-	LocalizationLoader = require('../../browser/LocalizationLoader'),
-	ServiceLocator = require('catberry-locator');
+const assert = require('assert');
+const LocalizationLoader = require('../../browser/LocalizationLoader');
+const ServiceLocator = require('catberry-locator');
 
-describe('client/LocalizationLoader', function () {
-	describe('#load', function () {
-		it('should return the same object from window on all specified locales',
-			function () {
-				var locator = new ServiceLocator(),
-					localization = {
-						TEST_VALUE: 'test'
-					};
+/* eslint max-nested-callbacks: [2, 4]*/
+
+describe('client/LocalizationLoader', () => {
+	describe('#load', () => {
```

Comment:
Hmm, interesting. Mocha maintainers say [that it does not work with arrow functions](https://mochajs.org/#arrow-functions). Are you sure that works?